packages/mgt-components/src/components/BasePersonCardSection.ts

```diff
@@ -41,7 +41,7 @@ export abstract class BasePersonCardSection extends MgtTemplatedComponent {
     }
 
     this._personDetails = value;
-    this.requestStateUpdate();
+    void this.requestStateUpdate();
```

Comment:
You'll see a lot of these as we have lots of calls to async methods in synchronous setters...